### DISCUSSION TITLE
Why does tar_visnetwork want to modify _targets.yaml ?

### DISCUSSION THREAD

**Discussion by Balaika** [ropensci/targets#768] [Help]
I am playing with 2 different _targets.yaml and I noticed that tar_visnetwork try to edit _targets.yaml.
Is it a bug ?
If I chmod a-w _targets.yaml, tar_visnetwork does not work anymore:

```
>tar_visnetwork()

Error in assign(".Traceback", traceback(9), envir = baseenv()) :
  cannot add binding of '.Traceback' to the base environment
In addition: Warning message:
In file(file, "w", encoding = fileEncoding) :
  cannot open file '_targets.yaml': Permission denied
```

**Comment by wlandau** [comment]
What version of `targets` are you using? Version 0.8.0 and above handles the yaml differently. I could not reproduce this error using the current development version.

**Reply by Balaika** [reply]
Ok this is my mistake. In the _targets.R file I left a tar_config_set(script=...) line, which is why a non editable _targets.yaml was a problem !
